src/Infrastructure/Event/Profiler/ProfilerEventListener.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Event\Profiler;
+
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ProfilerEventListener
```

Comment:
Unit test is not written because it uses xhprof functions specific only in local dev env